src/components/order-summary/order-summary.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="order-summary" data-js="order-summary">
+<div class="order-summary sticky-moneybox" data-js="order-summary">
```

Comment:
I would suggest to change `sticky-moneybox` class name to be more BEM (since it is a modifier). i.e. `order-summary--sticky-moneybox`.

Reply:
Yeah, make sense. I will make the change